app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -211,7 +211,8 @@ private void setCookieAndRecordMetrics(UserSession session) {
         // We have removed the cookie in the past, only to find out that clients were unknowingly
         // depending on the cookie to preserve the session token. So it remains.
         response().setCookie(BridgeConstants.SESSION_TOKEN_HEADER, session.getSessionToken(),
-                BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS, "/");
+                BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS, "/", 
+                bridgeConfig.getHostnameWithPostfix("webservices"), true, true);
```

Comment:
This doesn't work on local.

1. You want to use getHostnameWithPostfix() resolves to webservices-local.sagebridge.org, which doesn't resolve to anything. I think you want getWebservicesUrl().
2. Local doesn't use HTTPS. Can we set these flags based on bridgeConfig.getEnv() != LOCAL?

(Same as in BaseController.)

Reply:
Our system doesn't use cookies so it doesn't effect us, but I can change this for localhost. I would like to remove the cookie, but it apparently breaks some Android apps that have been built against the Bridge server.

Reply:
Changed this to not set the SSL-related flags to true in local environment